pysilcam/tests/test_background.py

```diff
@@ -1,27 +1,29 @@
 # -*- coding: utf-8 -*-
 import numpy as np
 from pysilcam.acquisition import Acquire
-from pysilcam.background import backgrounder
+from pysilcam.background import Backgrounder
 
 
 def test_background_aquire():
-    '''Testing background-corrected frame acquisition'''
+    bg = Backgrounder(5, None)
```

Comment:
F841: local variable 'bg' is assigned to but never used  <!-- Automatically posted by Lintly -->